pkg/doc/verifiable/support_test.go

```diff
@@ -202,3 +202,12 @@ func (s rs256TestSigner) Sign(data []byte) ([]byte, error) {
 
 	return rsa.SignPKCS1v15(rand.Reader, s.privKey, crypto.SHA256, hashed)
 }
+
+const (
+	rsaPublicKeyType     = "RSA"
+	ed25519PublicKeyType = "ed25519"
```

Comment:
probably expose these key types or better yet, use KeyType defined in pkg/kms/api.go as it should be the official key type in Aries-Framework-Go

and replace all references of "ed25519" in this patch.. there's at least 12, but in different packages than `verifiable`

Reply:
Makes sense! I added `PublicKeyType` in a way similar to `kms.KeyType`.